src/include/ipc4/base_fw.h

```diff
@@ -42,7 +42,7 @@
 #define IPC4_MAX_LIBS_COUNT 16
 
 /* Max pipeline count supported by ipc4 */
-#define IPC4_MAX_PPL_COUNT 16
+#define IPC4_MAX_PPL_COUNT 32
```

Comment:
@marcinszkudlinski @abonislawski @mwasko fyi - any issue here ?

Reply:
@lgirdwood looks good to me, I believe it should be 32 at a first place. @mmaka1?